generators/deployment/templates/pipeline_master.yml

```diff
@@ -0,0 +1,192 @@
+stages:
+- name: Build Stage
+  inputs:
+  - type: git
+    branch: master
+    service: ${REPO}
+{{#if triggersType}}
+  triggers:
+    - type: {{triggersType}}
+{{/if}}
+  jobs:
+  - name: Build
+    type: builder
```

Comment:
If deploy type is Kube should we run both a build and container build, or just a build?